### PR TITLE
DL-313: Rename topics sent to firefox

### PR DESCRIPTION
**Pull request by jonathankoren**
# Description

https://getpocket.atlassian.net/browse/DL-313

Adzerk is sending the wrong topic names through to firefox when they've been editorially chosen by the ads team. Instead of being "nb_topic_foo" they should simply be "foo". Additionally, the array of strings was changed to a map of strings to constant 1s for compatibility with the personality provider in Firefox. 

# Tests
Units

### CODE REVIEW

**Comment by jonathankoren** [comment]
@mmiermans don’t forget to merge this.

File: adzerk/transform.py [hunk 1]
```diff
@@ -116,7 +116,8 @@ def get_personalization_models(body):
     else:
         # Topics in AdZerk prefixed with topic_ correspond with models in Firefox prefixed with nb_model_.
```
Comment (ssami) [inline adzerk/transform.py:117]: Let's fix this comment too

Comment (mmiermans) [reply adzerk/transform.py:117]: I think this comment is still accurate, but there is logic in Firefox the prepends the `nb_model_` prefix, is my understanding.
https://searchfox.org/mozilla-central/source/browser/app/profile/firefox.js#1374

Comment (jonathankoren) [reply adzerk/transform.py:117]: The files downloaded from the sever are nb_model_foo.

Internally to those files are the tags. The tags are simply too. It is these tags that are used inside the logic.

**Pull request merged**